crates/orchestrator/src/generators/chain_spec.rs

```diff
@@ -1158,6 +1243,42 @@ mod tests {
         serde_json::from_str(&content).unwrap()
     }
 
+    #[test]
+    fn overrides_from_toml_works() {
+        use serde::{Deserialize, Serialize};
+
+        #[derive(Debug, Serialize, Deserialize)]
+        struct MockConfig {
+            #[serde(rename = "genesis", skip_serializing_if = "Option::is_none")]
+            genesis_overrides: Option<serde_json::Value>,
+        }
+
+        let mut chain_spec_json = chain_spec_test(ROCOCO_LOCAL_PLAIN_TESTING);
+        // Could also be  something like [genesis.runtimeGenesis.patch.balances]
```

Comment:
nit:
Can we also add an example for this in the test, besides the dev accounts, and assert on its existence like we did with `dev accounts`?

Reply:
Yes, I have a todo for adding more tests/examples. But I think is ok to unlock you asap.

Reply:
Sounds good! thx!